src/test/resources/test_yaml/android_full.yaml

```diff
@@ -16,6 +16,7 @@ defaults:
     platformVersion: 6.0
     deviceName: Pixelated Nexus
     avd: Nexus 13
+    avdArgs: -no-window
```

Comment:
Will this work with multiple specified arguments?

We should make a test for this scenario

Reply:
I have an alias in my bash profile that works with multiple args: `alias emulator='emulator @Nexus5x -noaudio -no-boot-anim'`

Reply:
This is something similar to what would be optimal on the ubuntu VM:
`emulator @generic_9.0 -no-window -no-boot-anim -netdelay none -no-snapshot -wipe-data -read-only -skin 480x800 &`

Originally I was thinking: Ideally we would be able to set each of these separate args as separate env variables in the project's env variables on TeamCity. That way the build configurations could drive what is being tested on rather than having to manually change it in the `config.yaml`. This would enable the ability to test on multiple emulators for the same project.

You're right though, perhaps it will work with just one env variable with the entire string. That would be dope. But also, why wouldn't just appending all of the args to the emulator name not work?

So let's make a test that avdArgs will work with something like `-no-window -no-boot-anim -netdelay none -no-snapshot -wipe-data -read-only -skin 480x800 &` as the parameter.

And I assume putting these args after Nexus 13 won't work, but can we try that? I wonder why it won't work

Reply:
The main goals are:
1) enable args to the emulator
2) allow these args to be changed without having to make changes to the entire repo so we can clone build configs to run tests against multiple api levels

Reply:
Nevermind about the avd name stuff, I just realized this is the Appium configuration http://appium.io/docs/en/writing-running-appium/caps/

But I guess it would still be nice to add the test for multiple to make sure it works, and also perhaps in a later PR we can see what can be done about the environment variable control of the emulator. For now the focus on getting these tests running is good.

Reply:
Will work on tests for multiple args.  Good suggestion, thanks!